utils/test_harness/sysman/include/test_harness_sysman_device.hpp

```diff
@@ -19,6 +19,13 @@ void sysman_device_reset(zes_device_handle_t device);
 zes_device_properties_t
 get_sysman_device_properties(zes_device_handle_t device);
 
+std::vector<zes_subdevice_exp_properties_t>
+get_sysman_subdevice_properties(zes_device_handle_t device, uint32_t *pCount);
+
+zes_device_handle_t get_device_by_uuid(zes_driver_handle_t hDriver,
```

Comment:
can we rename this function "get_sysman_device_by_uuid()"?

Reply:
Done.